android/fakewallet/src/main/java/com/solana/mobilewalletadapter/fakewallet/MobileWalletAdapterViewModel.kt

```diff
@@ -32,6 +28,9 @@ class MobileWalletAdapterViewModel(application: Application) : AndroidViewModel(
     val mobileWalletAdapterServiceEvents =
         _mobileWalletAdapterServiceEvents.asSharedFlow() // expose as event stream, rather than a stateful object
 
+    private val _lowPower = MutableSharedFlow<Boolean>(1)
```

Comment:
Rather than create a new flow for this, could you create another subclass of `MobileWalletAdapterServiceRequest`, representing the low-power/no-connection event?